### DISCUSSION TITLE
Exclusive locks are not supported, how to enable exclusive lock in termux?

### DISCUSSION THREAD

**Discussion by bhiantara** [termux/termux-app#2448] [Q&A] [answered]
i have problem with Exclusive Lock in termux storage access. How to enable exclusive lock in termux?

**Comment by agnostic-apollo** [comment, marked as answer]
What storage exclusive lock are you talking about?

**Reply by bhiantara** [reply]
> What storage exclusive lock are you talking about?

hi i tryed to run laravel project using termux, unfotunately have problem with storage exclusive lock does not supported , i found answered in stackoverflow it because vagrant access. The problem is in NFS options. By default NFS is using NLM locking, so how to change vagrant for storage access?

**Reply by bhiantara** [reply]
this is my reference from  stackoverflow: 
https://stackoverflow.com/a/35743592/2707570

**Reply by agnostic-apollo** [reply]
You will have to specific `nolock` when mounting the `nfs` storage and will need to run `mount` command in mount master (global namespace) for the mount point to be accessible to other apps. This is not a termux issue and nothing the app can do.

https://android.stackexchange.com/questions/200867/how-to-mount-nfs-on-android-with-correct-permissions

https://www.reddit.com/r/termux/comments/qzrjjy/comment/hloexgj
